src/DssSpell.sol

```diff
@@ -52,157 +63,97 @@ contract DssSpellAction is DssAction {
-        // London Business School Blockchain - 3798 DAI - 0xB83b3e9C8E3393889Afb272D354A7a3Bd1Fbcf5C
-        DssExecLib.sendPaymentFromSurplusBuffer(LBSBLOCKCHAIN,        3_798);
-        // CalBlockchain - 3421 DAI - 0x7AE109A63ff4DC852e063a673b40BED85D22E585
-        DssExecLib.sendPaymentFromSurplusBuffer(CALBLOCKCHAIN,        3_421);
-        // Blockchain@Columbia - 2851 DAI - 0xdC1F98682F4F8a5c6d54F345F448437b83f5E432
-        DssExecLib.sendPaymentFromSurplusBuffer(BLOCKCHAINCOLUMBIA,   2_851);
-        // Frontier Research LLC - 2285 DAI - 0xA2d55b89654079987CF3985aEff5A7Bd44DA15A8
-        DssExecLib.sendPaymentFromSurplusBuffer(FRONTIERRESEARCH,     2_285);
-        // Chris Blec - 1334 DAI - 0xa3f0AbB4Ba74512b5a736C5759446e9B50FDA170
-        DssExecLib.sendPaymentFromSurplusBuffer(CHRISBLEC,            1_334);
-        // CodeKnight - 355 DAI - 0x46dFcBc2aFD5DD8789Ef0737fEdb03489D33c428
-        DssExecLib.sendPaymentFromSurplusBuffer(CODEKNIGHT,             355);
-        // ONESTONE - 342 DAI - 0x4eFb12d515801eCfa3Be456B5F348D3CD68f9E8a
-        DssExecLib.sendPaymentFromSurplusBuffer(ONESTONE,               342);
-        // pvl - 56 DAI - 0x6ebB1A9031177208A4CA50164206BF2Fa5ff7416
-        DssExecLib.sendPaymentFromSurplusBuffer(PVL,                     56);
-        // ConsenSys - 33 DAI - 0xE78658A8acfE982Fde841abb008e57e6545e38b3
-        DssExecLib.sendPaymentFromSurplusBuffer(CONSENSYS,               33);
+        // MOMC Parameter Changes
+        // https://vote.makerdao.com/polling/QmYUi9Tk
+
+        // Increase WSTETH-B Stability Fee to 0.25%
+        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_PT_TWO_FIVE_PCT_RATE, true);
+
+        // Increase Compound v2 D3M Maximum Debt Ceiling to 20 million
+        // Set Compound v2 D3M Target Available Debt to 5 million DAI (this might already be the case)
+        DssExecLib.setIlkAutoLineParameters("DIRECT-COMPV2-DAI", 20 * MILLION, 5 * MILLION, 12 hours);
+
+        // Increase the USDP PSM tin to 0.2%
+        DssExecLib.setValue(MCD_PSM_PAX_A, "tin", 20 * WAD / 10000);   // 20 BPS
+
+
+        // MKR Transfer for CES
+        // https://vote.makerdao.com/polling/QmbNVQ1E#poll-detail
+
+        // CES-001 - 96.15 MKR - 0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da
+        GemLike(DssExecLib.mkr()).transfer(CES_WALLET, 96.15 ether); // ether as solidity alias
+
+
+        // Cage DIRECT-AAVEV2-DAI
+        // https://forum.makerdao.com/t/housekeeping-tasks-for-next-executive/19472
+
+        // Cage DIRECT-AAVEV2-DAI to prepare for new deployment
+        //
+        CageLike(MCD_JOIN_DIRECT_AAVEV2_DAI).cage();
+        bytes32 _ilk = "DIRECT-AAVEV2-DAI";
+        DssExecLib.removeIlkFromAutoLine(_ilk);
+        (,,, uint256 _line,) = VAT.ilks(_ilk);
+        DssExecLib.setValue(address(VAT), _ilk, "line", 0);
+        DssExecLib.setValue(address(VAT), "Line", VAT.Line() - _line);
+        DssExecLib.setValue(MCD_CLIP_DIRECT_AAVEV2_DAI, "stopped", 3);
```

Comment:
since it never was liquidatable, should we just remove this and leave the test as is to show it is what we want it to be before and after casting?

Reply:
Part of me wants to remove this for efficiency but also we may want to keep it as a pattern for future offboardings.